package.json

```diff
@@ -46,7 +46,8 @@
     "tsc": "tsc -p ./tsconfig.json --noEmit --pretty --skipLibCheck",
     "tsc:cypress": "tsc -p ./cypress/tsconfig.json --noEmit --pretty --skipLibCheck",
     "cy": "CYPRESS_baseUrl=http://localhost:9999 cypress open",
-    "cy:dev": "CYPRESS_baseUrl=http://kubernetes.docker.internal:8080 cypress open",
+    "cy:dev": "source ../monitor-ci/.env && CYPRESS_baseUrl=http://kubernetes.docker.internal:$PORT_HTTPS cypress open --config testFiles=shared/**/*.*",
```

Comment:
specifying the testFiles here so that the default command only tests the shared tests